app/lib/common/dialogs/attachment_selection.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:io';
```

Comment:
I _assume_ this is mostly copy-paste from before ...

Reply:
Logically same, with the migration of some code to subsequent modals of it.